### ISSUE TITLE
How to convert the trained model in MatConvNet to the tensorflow model?

### ISSUE THREAD

**Issue by pmann88** [vlfeat/matconvnet#1021]
I am wondering if it is possible to convert the MatConvNet model to Tensorflow for the deployment purpose?

**Comment by nmerrill67** [comment]
There is a [https://github.com/vlfeat/matconvnet/blob/master/utils/simplenn_caffe_deploy.m](script) for converting a simplenn to caffe model, and there is another [program](https://github.com/ethereon/caffe-tensorflow) for caffe to TF model. Will it  have the same accuracy? There is only one way to find out. Also, it does not work for dagnn.

**Comment by melissadale** [comment]
I had a great deal of difficulty converting a MatConvNet model to Caffe (mostly because getting Caffe properly working on my system proved challenging). My MatConvNet was very small, and pretty basic as far as CNN architectures go. So I was able to manually pull the trained weights/biases, and other data (e.g. the learning rate, weight decay, etc) from the MatConvNet mat file. I cannot speak to how accuracies compare between models, as I am currently working with unlabeled data, but visually looking at the output, it appears to be consistent. 

**tl;dr** I hobbled together a conversion system that seems to work for my small application. I shared my code in case you can alter it for your needs here: https://github.com/melissadale/MatConv2TensorFlow
